src/cpu/xbyak/xbyak.h

```diff
@@ -242,7 +242,7 @@ class Error : public std::exception {
 	{
 		if (err_ < 0 || err_ > ERR_INTERNAL) {
 			fprintf(stderr, "bad err=%d in Xbyak::Error\n", err_);
-			exit(1);
```

Comment:
Please take this change to Xbyak if this change is absolutely necessary.

Reply:
Yes, this is absolutely necessary. Otherwise, MKL-DNN and other libraries that link it statically will be rejected from Fedora and other distributions. We're not using Xbyak directly, so please take care of taking this to Xbyak if you think it's necessary, but since MKL-DNN uses a copy of it, I don't see why this can't be merged immediately.

Reply:
We can handle talking to Herumi, but we are not merging this until we have a solution that works for Xbyak. We are keeping an exact copy of Xbyak to make merges easy. While I agree with the 'no-exits statement', this is a corner case and exit is never called in practice.

Reply:
Thanks for taking this issue upstream, but I disagree that this is corner case. If the library links exit, than in practice it *can* be called, and this disqualifies it automatically from Fedora and potentially other distributions because it fails with their testing tools. Open Image Denoise was rejected as well because of this issue in MKL-DNN.

Reply:
I understand Fedora's stance, and I am sorry that that Open Image Denoise was rejected, but this  does not sound like a reason for this change to go around the rules we have for interacting with Xbyak upstream. I've opened the issue here: https://github.com/herumi/xbyak/issues/83

Reply:
The changes are implemented upstream. We will pull this in shortly.

Reply:
The change is promoted in f32e21b48948d4bc5402b77e2f6bd99f2eaa76ea.